out/css/wirecard_wdoxidee_common.css

```
@@ -0,0 +1,18 @@
+.wdoxidee-messagebox {
+  border-radius: 4px;
+  margin-bottom: 20px;
+  padding: 15px;
+}
+
+.wdoxidee-info,
```

Do we write our custom CSS using BEM? If so, this should be:
```suggestion
.wdoxidee-messagebox--info,
```

---

👍